src/applications/personalization/profile-2/tests/e2e/profile.cypress.spec.js

```diff
@@ -0,0 +1,105 @@
+import { PROFILE_PATHS } from '../../constants';
+
+import mockUser from '../fixtures/users/user-36.json';
+import mockPaymentInfo from '../fixtures/payment-information/direct-deposit-is-set-up.json';
+import mockFeatureToggles from '../fixtures/feature-toggles.json';
```

Comment:
We have a `cy.syncFixtures` [custom command](https://github.com/department-of-veterans-affairs/va.gov-team/blob/6dccdd3ac61d7fbfa8b30e317230cc9212a7c9f8/teams/vsp/teams/tools/frontend/cypress-migration-guide.md#test-data-fixtures-) if you'd like to be able to reference fixtures from designated folders without individually importing each one.

Reply:
I looked at that briefly and will probably switch to using that if I add more test cases.

Reply:
```js
cy.syncFixtures({
  fixtures: path.join(__dirname, '..', 'fixtures'),
});

...

cy.route('GET', '/v0/user', 'fx:fixtures/user-36');
```

Reply:
I might be doing something wrong, but I'm getting this:
```
cy.task('_syncFixtures') failed with the following error:

> Fixture path "fixtures" conflicts with another fixture path at "fixtures"
```

Reply:
Hm... `synxFixtures` essentially works by creating a path under `cypress/fixtures/tmp` and uses symlinks for the paths that you specify in the argument. So it sounds like there might be something creating `cypress/fixtures/tmp/fixtures` before it tries to link that to the directory you're assigning to `fixtures`.

Basically there could be an overlap in the folder structure if you're specifying multiple fixtures in the command.